### DISCUSSION TITLE
Customizing Shell Tabbar

### DISCUSSION THREAD

**Discussion by StephanZilke** [dotnet/maui#11649] [Q&A]
Is there any possibility to customize the appearance of the Shell Tabs? If a tab is selected, I want to show an indicator (e.g. a line above the icon), How can this be achieved?

**Comment by VladislavAntonyuk** [comment]
It could be a good start point for you: https://vladislavantonyuk.github.io/articles/Customizing-.NET-MAUI-Shell

**Reply by JoacimWall** [reply]
Hi 
I have looked on you samples and tried to modify to change the height and add rounded corners. It's works on Android but on iOS I am not able to change the height of the control.  Do you know if it's possible to change height on tabbar in shell?

``
public override void SetAppearance(UITabBarController controller, ShellAppearance appearance)
    {
        base.SetAppearance(controller, appearance);
       
        controller.TabBar.AutoresizingMask = UIViewAutoresizing.None;
       var newHeight = 200; 
       var tabBarFrame = controller.TabBar.Frame;
       tabBarFrame.Height = tabBarFrame.Height + newHeight;
       tabBarFrame.Y = tabBarFrame.Y - newHeight;
       controller.TabBar.Frame = tabBarFrame;
       
       UIBezierPath uIBezierPath = UIBezierPath.FromRoundedRect(controller.TabBar.Bounds, UIRectCorner.TopLeft | UIRectCorner.TopRight, new CoreGraphics.CGSize(20, 20));
       CAShapeLayer cAShapeLayer = new CAShapeLayer();
       cAShapeLayer.Frame =controller.TabBar.Bounds; // new CGRect(controller.TabBar.Bounds.X,controller.TabBar.Bounds.Y-50,controller.TabBar.Bounds.Width, controller.TabBar.Bounds.Height+150) ;
       cAShapeLayer.Path = uIBezierPath.CGPath;
       controller.TabBar.Layer.Mask = cAShapeLayer;  
      
    }
``
